github/github.go

```diff
@@ -144,6 +147,11 @@ type UploadOptions struct {
 	Name string `url:"name,omitempty"`
 }
 
+// BreakdownOptions specifies the parameters to methods that support breakdown per day or week.
+type BreakdownOptions struct {
```

Comment:
Usually these options are scoped out to their method if only used once, e.g. TrafficBreakdownOptions. See OrgMembershipOptions, for example. But I defer to the real maintainers of this project for a second. :smile: